README.md

```diff
@@ -108,7 +108,7 @@ Read the following article to learn how TestCafe Studio could fit into your work
 
 ### Installation
 
-Ensure that [Node.js](https://nodejs.org/) (version 6 or newer) and [npm](https://www.npmjs.com/) are installed on your computer before running it:
+Ensure that [Node.js](https://nodejs.org/) ([Current or Active LTS](https://github.com/nodejs/Release#release-phases)) and [npm](https://www.npmjs.com/) are installed on your computer before running it:
```

Comment:
Current is 15 version, Active LTS is 14 (https://nodejs.org/en/about/releases/).

TestCafe supports the Node.js version starting from 10 (https://github.com/DevExpress/testcafe/blob/master/package.json#L19).

Let's specify the minimal supported version - 10.

Reply:
Corrected to this:

Ensure that [Node.js](https://nodejs.org/) ([Current or Active LTS](https://github.com/nodejs/Release#release-phases) is recommended, version 10 at minimum) and [npm](https://www.npmjs.com/) are installed on your computer before running it.

Is it ok now?

Reply:
_(just to clarify -- I wanted to avoid specifying particular versions, so we don't need to update the main page each time the particular version dies like version 6)_